### PR TITLE
Add pagesize in configuration

### PR DESCRIPTION
**Pull request by pecorarista**

### CODE REVIEW

File: app/controllers/Application.scala [hunk 1]
```diff
@@ -50,6 +51,8 @@ class Application @Inject() (
 
   implicit val timeout = 10.seconds
 
+  lazy val config = ConfigFactory.load()
```
Comment (satendrakumar06) [inline app/controllers/Application.scala]: Use play configuration manager instead of ConfigFactory.

**Comment by satendrakumar06** [comment]
@pecorarista Thanks For PR. Please see my comment.

**Comment by pecorarista** [comment]
@satendrakumar06 Thank you for your comment. I fixed it. Is this all right?

**Comment by satendrakumar06** [comment]
LGTM

**Pull request merged**